### ISSUE TITLE
Doubts about the issue of automatic focusing

### ISSUE THREAD

**Issue by SuZhelevel6** [CocoCR300/flauncher#40]
This is based on remote control operation：

When I turned on the   `automatically hide status bar` function, I found this problem:
When I want to move from the APP Card on the left（For example, the first app） to the settings button, I will press the up button.
But at this moment, it did not immediately focus on the settings button.
At this moment, it seems that it is not focused on any components, or on components that are not visible.
I need to press an additional right-click button to focus on the settings button

This situation also occurs in many settings items on the settings interface:
1. `Applications`  We need to press the down button or right button
2. `Status bar`  We need to press the down button 
3. `Back button action `   We need to press the down button 
...

But in `Wallpaper Settings` it can Immediately focus on `Gradient` 

Although this situation is not a big issue in terms of user experience, I still want to know if anyone knows why there is such a difference?

**Comment by CocoCR300** [comment]
As for the status bar, I just discovered the problem, it will be fixed soon.
I got used to the focus behaving that way, but it can be easily fixed. I can't seem to fix the last of them, and there's one place in which it is better for it to stay that way, the side panel that appears when you long press an application, if the first item there were to be focused automatically, it would be selected immediately, at least that happened when I was testing it with a keyboard a while ago.
